app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -112,37 +112,38 @@ class BookmarksActivity : DuckDuckGoActivity() {
                     is BookmarksViewModel.Command.ConfirmDeleteSavedSite -> confirmDeleteSavedSite(it.savedSite)
                     is BookmarksViewModel.Command.OpenSavedSite -> openSavedSite(it.savedSite)
                     is BookmarksViewModel.Command.ShowEditSavedSite -> showEditSavedSiteDialog(it.savedSite)
-                    is BookmarksViewModel.Command.ImportedBookmarks -> showImportedBookmarks(it.importBookmarksResult)
-                    is BookmarksViewModel.Command.ExportedBookmarks -> showExportedBookmarks(it.exportBookmarksResult)
+                    is BookmarksViewModel.Command.ImportedSavedSites -> showImportedSavedSites(it.importSavedSitesResult)
+                    is BookmarksViewModel.Command.ExportedSavedSites -> showExportedSavedSites(it.exportSavedSitesResult)
                 }
             }
         )
     }
 
-    private fun showImportedBookmarks(result: ImportBookmarksResult) {
+    private fun showImportedSavedSites(result: ImportSavedSitesResult) {
         when (result) {
-            is ImportBookmarksResult.Error -> {
+            is ImportSavedSitesResult.Error -> {
                 showMessage(getString(R.string.importBookmarksError))
             }
-            is ImportBookmarksResult.Success -> {
-                if (result.bookmarks.isEmpty()) {
+            is ImportSavedSitesResult.Success -> {
+                if (result.savedSites.isEmpty()) {
                     showMessage(getString(R.string.importBookmarksEmpty))
                 } else {
-                    showMessage(getString(R.string.importBookmarksSuccess, result.bookmarks.size))
+                    // do we want different messages based on having imported favorites or bookmarks only or both?
```

Comment:
I think we can remove this comment since we will release showing the same message.